diesel/src/query_builder/update_statement/mod.rs

```diff
@@ -10,7 +10,7 @@ use query_builder::{Query, AsQuery, QueryFragment, QueryBuilder, BuildQueryResul
 use query_source::Table;
 use result::QueryResult;
 
-/// The type returned by [`update`](fn.update.html). The only thing you can do
+/// The type returned by [`update`](/diesel/fn.update.html). The only thing you can do
```

Comment:
Same nit here on absolute links. Although this definitely makes me think we need a way to say "root of the docs for the current version" easily.